daemon/container.go

```diff
@@ -1036,6 +1036,9 @@ func (container *Container) networkMounts() []execdriver.Mount {
 }
 
 func (container *Container) addLocalMountPoint(name, destination string, rw bool) {
+	if container.MountPoints == nil {
+		container.MountPoints = map[string]*mountPoint{}
```

Comment:
Not sure if we discussed this, but why not instantiate it on Container struct creation?